team_test.go

```diff
@@ -69,14 +69,250 @@ func TestGetTeamWithAlias(t *testing.T) {
+        "teams": {
+          "nodes": [
+            {
+              "alias": "security",
+              "aliases": [
+                "security"
+              ],
+              "contacts": [],
+              "htmlUrl": "https://app.opslevel.com/teams/security",
+              "id": "Z2lkOi8vb3BzbGV2ZWwvVGVhbS84NTE",
+              "manager": {
+                "name": "Kyle Rockman",
+                "email": "kyle.rockman@mac.com"
+              },
+              "members": {
+                "nodes": [
+                  {
+                    "name": "Andrew Example",
+                    "email": "example@opslevel.com"
+                  },
+                  {
+                    "name": "Ashley Example",
+                    "email": "example@opslevel.com"
+                  },
+                  {
+                    "name": "Drew Example",
+                    "email": "example@opslevel.com"
+                  }
+                ],
+                "pageInfo": {
+                  "hasNextPage": false,
+                  "hasPreviousPage": false,
+                  "startCursor": "MQ",
+                  "endCursor": "Mw"
+                }
+              },
+              "name": "Security",
+              "responsibilities": null
+            },
+            {
+              "alias": "vps",
+              "aliases": [
+                "vps"
+              ],
+              "contacts": [],
+              "htmlUrl": "https://app.opslevel.com/teams/vps",
+              "id": "Z2lkOi8vb3BzbGV2ZWwvVGVhbS84NTI",
+              "manager": {
+                "name": "Kyle Rockman",
+                "email": "kyle.rockman@mac.com"
+              },
+              "members": {
+                "nodes": [
+                  {
+                    "name": "Andrew Example",
+                    "email": "example@opslevel.com"
+                  },
+                  {
+                    "name": "Ashley Example",
+                    "email": "example@opslevel.com"
+                  },
+                  {
+                    "name": "Drew Example",
+                    "email": "example@opslevel.com"
+                  }
+                ],
+                "pageInfo": {
+                  "hasNextPage": false,
+                  "hasPreviousPage": false,
+                  "startCursor": "MQ",
+                  "endCursor": "Mw"
+                }
+              },
+              "name": "VPs",
+              "responsibilities": null
+            }
+          ],
+          "pageInfo": {
+            "hasNextPage": false,
+            "hasPreviousPage": true,
+            "startCursor": "NDc",
+            "endCursor": "EOf"
+          }
+        }
+      }
+    }
+  }`},
+	}
+	client := APaginatedTestClient(t, "team/list", requests...)
 	// Act
-	result, err := client.ListTeams()
+	response, err := client.ListTeams(nil)
+	result := response.Nodes
 	// Assert
 	autopilot.Ok(t, err)
 	autopilot.Equals(t, 5, len(result))
```

Comment:
user total count here

Reply:
Did you mean team total count? I changed it to team total count but I can change it to user total count if that's better.

Reply:
sorry use i mean "use total count here"